core/orchestrator.go

```diff
@@ -141,17 +141,28 @@ func (orch *orchestrator) ProcessPayment(payment net.Payment, manifestID Manifes
 			}
 		}
 
+		// Add ticket EV to credit
+		orch.node.Accounts.Credit(manifestID, new(big.Rat).Mul(new(big.Rat).SetFrac(ticket.FaceValue, big.NewInt(1)), ticket.WinProbRat()))
```

Comment:
We can use [(*Ticket).EV()](https://github.com/livepeer/go-livepeer/blob/master/pm/ticket.go#L80) here.

Reply:
Worth rebasing with master to pull in the changes from #937 - we should only credit the ticket EV to B's balance if there was either no error receiving the ticket or if the error receiving the error was acceptable (see this [TODO](https://github.com/livepeer/go-livepeer/pull/937/files#diff-b76fbd82bbf5b7a4d2fdf59ecd8d4ca6R147)).

Reply:
Fixed in https://github.com/livepeer/go-livepeer/pull/956/commits/32c2ccb4c36e850fa79e065f2566f43b6721550b